website/src/documentation/programming-guide.md

```diff
@@ -39,6 +39,9 @@ how to implement Beam concepts in your pipelines.
   </ul>
 </nav>
 
+{:.language-py}
+The Python SDK supports Python 2.7, 3.5, or higher; soon, new Python SDK releases won't support Python 2.7 ([BEAM-8371](https://issues.apache.org/jira/browse/BEAM-8371)). For best results, use Beam with Python 3.
```

Comment:
Should we spell out `3.5, or higher` here similarly as `3.5, 3.6 and 3.7`?

Reply:
SGTM, would not hurt. We should have a conversation in Beam about definition of support for minor version. Direct runner might just work on 3.8, dataflow wouldn't. There are no 3.8 tests. Do we want to stop enthusiastic users from trying 3.8? Probably not. Do we claim to support it? Probably not.

Reply:
Done